modules/storageFront.js

```diff
@@ -64,7 +65,11 @@ GM_ScriptStorageFront.prototype.getValue = function(name, defVal) {
   if (value === undefined || value === null) return defVal;
 
   try {
-    return JSON.parse(value);
+    value = JSON.parse(value);
+    // JSON (JavaScript Object Notation) is not designed for serializing
+    // DOM Nodes. But maybe in the future...
```

Comment:
What does this comment mean?  When would we ever want a DOM node in context of a storage call?